content/en/cosign/verify.md

```diff
@@ -9,8 +9,28 @@ The general verification format with the `cosign verify` command is as follows.
 ```shell
 $ cosign verify --key <key path>|<key url>|<kms uri> <image uri> 
 ```
+## Keyless verification using OpenID Connect
 
-We'll use `user/demo` as our example image in the following commands. 
+We'll use `user/demo` as our example image in the following commands and keyless signing where appropriate.
+
+To verify a container image keylessly, use the following command:
+
+```
+$ cosign verify <image URI> --certificate-identity=name@example.com 
+                            --certificate-oidc-issuer=https://accounts.example.com
+                           
+```
+
+The oidc-issuer for Google is https://accounts.google.com, Microsoft is https://login.microsoftonline.com, and GitHub is https://github.com/login/oauth.
+
+The following example verifies the signature on file.txt from user name@example.com issued by accounts@example.com. It uses a provided bundle cosign.bundle that contains the certificate and signature.
+
+```
+$ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@example.com 
+                              --certificate-oidc-issuer=https://accounts.example.com
+```
+
+With container images, the signature and certificate are attached to the container.  For blobs, the signature and certificate can be stored in a bundle file that is created at the time of signing.  Either it must be specified, or the individual signature and certificate must be specified.
```

Comment:
"Either it must be specified" --- can you replace "it" with the appropriate noun?